apps/web/src/pages/ApplicantDashboardPage/components/QualifiedRecruitments/QualifiedRecruitmentCard.tsx

```diff
@@ -0,0 +1,417 @@
+    if (!suspendedAt) {
+      return (
+        <div data-h2-display="base(flex)" data-h2-align-items="base(center)">
+          <span
+            data-h2-padding="base(0, x.5, 0, 0)"
+            data-h2-color="base(green)"
+          >
+            <CheckCircleIcon
+              data-h2-height="base(1em)"
+              data-h2-width="base(1em)"
+            />
+          </span>
+          <p>
+            {intl.formatMessage({
+              defaultMessage:
+                "You are actively appearing in talent results for this recruitment.",
+              id: "TDikDt",
+              description:
+                "Not suspended recruitment status message on qualified recruitment card.",
+            })}
+            {/* Dialog that allows user to suspend the qualified recruitment (application). Sets suspendedAt date to the current time. */}
+            <QualifiedRecruitmentDialog
+              id={id}
+              isSuspended={!!suspendedAt}
+              closeButtonLabel={intl.formatMessage({
+                defaultMessage: "Yes, remove me from search results",
+                id: "wgRXhR",
+                description:
+                  "Dialog close button on change search results status from qualified recruitment section.",
+              })}
+              openDialogLabel={intl.formatMessage({
+                defaultMessage: "Remove me",
+                id: "2g1mfB",
+                description:
+                  "Open dialog text on change search results status from qualified recruitment section.",
+              })}
+              primaryBodyText={intl.formatMessage({
+                defaultMessage:
+                  "You are currently appearing in talent search results for the “IT-01 Helpdesk support technician” pool.",
+                id: "dPoE9z",
+                description:
+                  "Dialog main body on change search results status from qualified recruitment section.",
+              })}
+              secondaryBodyText={intl.formatMessage({
+                defaultMessage:
+                  "If you’ve recently been placed or simply no longer want to be considered for opportunities related to this role, you can remove yourself from the list of candidates available for hire. This will NOT remove you from the recruitment itself and you can always re-enable your availability if you change your mind.",
+                id: "EWB2LC",
+                description:
+                  "Dialog main body on change search results status from qualified recruitment section.",
+              })}
+              title={intl.formatMessage({
+                defaultMessage: "Change your appearance in search results",
+                id: "FGF3sd",
+                description:
+                  "Dialog title on change search results status from qualified recruitment section.",
+              })}
+            />
+            .
+          </p>
+        </div>
+      );
+    }
```

Comment:
This may be better as its own component

Reply:
Good Idea! Fixed by [2dc20b0](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5988/commits/2dc20b08b77d936b98b13d28d756181c31e37b9a)